### ISSUE TITLE
Advanced Search "File Type" Hint Incorrect

### ISSUE THREAD

**Issue by shlake** [IQSS/dataverse#6189]
The hover hint for Advanced Search "FileType" is incorrect. It says "the extension", but it's not - it is the text that is in the "File Type" field. 
![Screen Shot 2019-09-19 at 10 12 47 AM](https://user-images.githubusercontent.com/3809325/65252302-b796bd80-dac6-11e9-9d10-756c6dd61d47.png)

The example gives "CSV", but one cannot search for "CSV" (returns no results), because the File Type is "Comma Separated Values" - so searching for "Comma" in Advanced FileType gives you all the CSV files.

Am suggesting to either fix the field search hint for "File Type". Some extensions "work", such as JPEG because the file type is "JPEG Image"  (but searching for the extension "JPG" does not work).

**Comment by pdurbin** [comment]
Just this week I was helping @atrisovic find all the R files in Harvard Dataverse with this query:

https://dataverse.harvard.edu/api/search?q=fileContentType%3Atype%2Fx-r-syntax&type=file

The problem @shlake has identified (thanks!) is the tip of an iceberg and I'd like us to consider fixing up searching for files somewhat holistically. Maybe we can get @TaniaSchlatter and @mheppler to weigh in.

In short, if you wrote a lot of the search code like I did, you know the tricks to do the searches you want. If you're not me, you might struggle a bit to find the files you want even with the Advanced Search page.
